nvaccess/nvda
# Disable WASAPI by default

### Link to issue number:
Changes to handle #15150
Follow up to #14697, #14896, #15038, #15097, #15145

### Summary of the issue:
WASAPI usage is known to cause intermittent crashing in #15150.
Generally, WASAPI code has not been proven to be stable.
Due to this, it should not be enabled by default in 2023.2.
WASAPI can be re-enabled by default once it is proven to be stable.

### Description of user facing changes
Disable WASAPI by default, preventing intermittent crashing #15150

### Description of development approach
Turn the WASAPI checkbox into a feature flag, so that it can easily be re-enabled in future.

### Testing strategy:
**Manual testing**
Upgrading the profile: Test starting NVDA with the WASAPI config value set to "True/False" instead of a "enabled/disabled/default".

Test the various controls related to WASAPI - ensure they are saved, applied and respected correctly.

### Known issues with pull request:
None

### Change log entries:
Refer to diff

### Code Review Checklist:



  - description is up to date
  - change log entries
  - Unit tests
  - System (end to end) tests
  - Manual testing
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
